documentation/source/QuickstartBasicToolkit.md

```diff
@@ -293,7 +293,7 @@ plt.imshow(image)
     export PYTHONPATH=$PYTHONPATH:<YOUR-LOCAL-PATH>/super-gradients/
     ```
 
-1. Launch one of SG's <a href="https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/recipes/Training_Recipes.md">training recipes</a>. For example, Resnet18 on Cifar10:
+1. Launch one of SG's <a href="https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/recipes">training recipes</a>. For example, Resnet18 on Cifar10:
```

Comment:
https://github.com/Deci-AI/super-gradients/blob/43ee312249581f07d22431ef5a12c6864aca16de/src/super_gradients/common/registry/README.md this file still has link to Training_Recipes.md

Reply:
Looks like the link was actually completely wrong to begin with, I fixed it here 78a25a71cf96c8508791e7ad7c0f760b6347d515